source/frontend/src/features/leases/detail/LeasePages/payment/table/periods/paymentColumns.tsx

```diff
@@ -21,222 +21,13 @@ import { Claims } from '@/constants';
 import { LeasePeriodStatusTypes } from '@/constants/leaseStatusTypes';
 import useKeycloakWrapper from '@/hooks/useKeycloakWrapper';
 import { ApiGen_CodeTypes_LeaseLicenceTypes } from '@/models/api/generated/ApiGen_CodeTypes_LeaseLicenceTypes';
-import { ApiGen_CodeTypes_LeasePaymentCategoryTypes } from '@/models/api/generated/ApiGen_CodeTypes_LeasePaymentCategoryTypes';
 import { ISystemConstant } from '@/store/slices/systemConstants';
 import { NumberFieldValue } from '@/typings/NumberFieldValue';
 import { prettyFormatDate } from '@/utils';
 import { stringToFragment } from '@/utils/columnUtils';
 import { formatMoney } from '@/utils/numberFormatUtils';
 
-import { FormLeasePeriod, FormLeasePeriodWithCategory } from '../../models';
-
-function getPeriodName({ row: { index, original } }: CellProps<FormLeasePeriod, unknown>) {
-  return (
-    <InlineFlexDiv className="align-items-center">
-      <b>{stringToFragment(`Period ${index + 1}`)}</b>&nbsp;
-      {original?.isVariable === 'true' && (
-        <TooltipWrapper tooltipId="variable-period-tooltip" tooltip="Variable Payments">
-          <StyledRefreshIcon size={16} title="Variable Payments" />
-        </TooltipWrapper>
-      )}
-    </InlineFlexDiv>
-  );
-}
-
-function startAndEndDate({ row: { original } }: CellProps<FormLeasePeriod, string>) {
-  return original.isFlexible === 'true' ? (
-    <>
-      {prettyFormatDate(original.startDate)} - {prettyFormatDate(original.expiryDate)}
-      <StyledBreak />
-      <i>(anticipated)</i>
-      <TooltipWrapper tooltipId="flexible-period-icon" tooltip="Flexible Period">
-        <StyledFlexibleIcon size={24} title="Flexible Period" />
-      </TooltipWrapper>
-    </>
-  ) : (
-    stringToFragment(
-      `${prettyFormatDate(original.startDate)} - ${prettyFormatDate(original.expiryDate)}`,
-    )
-  );
-}
-
-const renderExpectedTotal = () =>
-  function ({
-    row: { original },
-  }: CellProps<FormLeasePeriod, string> | CellProps<FormLeasePeriodWithCategory, string>) {
-    return stringToFragment(
-      original.paymentAmount !== undefined
-        ? formatMoney((original.paymentAmount as number) + ((original?.gstAmount as number) ?? 0))
-        : '-',
-    );
-  };
-
-function renderGstAmount({ row: { original } }: CellProps<FormLeasePeriod, NumberFieldValue>) {
-  return stringToFragment(
-    original.isGstEligible === true ? formatMoney(original?.gstAmount ?? 0) : '-',
-  );
-}
-
-function renderActualTotal({ row: { original } }: CellProps<FormLeasePeriodWithCategory, string>) {
-  const total = formatMoney(
-    (original.payments ?? [])
-      .filter(p => p.leasePaymentCategoryTypeCode?.id === original.category)
```

Comment:
This is where the actual bug was. This function was used for calculating the totals for the top table (period) and the category split. Because the top table did not have a category it did not add any of the payments.